### PR TITLE
fix ddl: TODO remove update cuz we need to handle it once

### PR DESCRIPTION
**Pull request by ice-cronus**

### CODE REVIEW

File: users/DDL.sql [hunk 1]
```diff
@@ -50,7 +50,13 @@ ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_step_blocked smallint NOT NULL DE
 ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_steps_last_updated_at timestamp[];
 ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_steps_created_at timestamp[];
 ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_user_id text;
+ALTER TABLE users ALTER COLUMN telegram_user_id SET DEFAULT '';
```
Comment (ice-ares) [inline users/DDL.sql]: U dont need this

File: users/DDL.sql [hunk 2]
```diff
@@ -50,7 +50,13 @@ ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_step_blocked smallint NOT NULL DE
...
+UPDATE users SET telegram_user_id = id WHERE telegram_user_id IS NULL;
```
Comment (ice-ares) [inline users/DDL.sql:53]: How can it be null since it always was not null?

File: users/DDL.sql [hunk 3]
```diff
@@ -50,7 +50,13 @@ ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_step_blocked smallint NOT NULL DE
...
+ALTER TABLE users ALTER COLUMN telegram_user_id SET NOT NULL;
```
Comment (ice-ares) [inline users/DDL.sql]: Its already not null

File: users/DDL.sql [hunk 4]
```diff
@@ -50,7 +50,13 @@ ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_step_blocked smallint NOT NULL DE
...
 ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_bot_id text;
+ALTER TABLE users ALTER COLUMN telegram_bot_id SET DEFAULT '';
```
Comment (ice-ares) [inline users/DDL.sql]: U dont need this

File: users/DDL.sql (hunk 2 above)
Comment (ice-cronus) [reply users/DDL.sql:53]: cuz we already has values with null, when I try to update not null I am getting an error

Comment (ice-cronus) [reply users/DDL.sql:53]: u're right, another column

**Pull request merged**